### ISSUE TITLE
Carthage crashes when bootstrapping.

### ISSUE THREAD

**Issue by guidomb** [Carthage/Carthage#776]
Carthage crashes while executing bootstrap with the [current version of master](https://github.com/Carthage/Carthage/commit/52249f46f370d8fbb82fdd71fe8b043b3b625061). All test pass in my local machine but after running `make install` and `carthage bootstrap --platform ios` in a project with the following `Cartfile` and `Cartfile.private` I get a huge crash.

**Cartfile:**
```
github "ReactiveCocoa/ReactiveCocoa" ~> 3.0.0
github "CocoaLumberjack/CocoaLumberjack" ~> 2.0.1
github "Wolox/WLXBluetoothDevice" ~> 0.2.0
```

**Cartfile.private:**
```
github "specta/specta" ~> 1.0.0
github "specta/expecta" ~> 1.0.1
```

**Cartfile.resolved:**
```
github "robrix/Box" "1.2.2"
github "CocoaLumberjack/CocoaLumberjack" "2.0.1"
github "specta/expecta" "v1.0.3"
github "specta/specta" "v1.0.3"
github "antitypical/Result" "0.4.4"
github "ReactiveCocoa/ReactiveCocoa" "v3.0.0"
github "Wolox/WLXBluetoothDevice" "0.2.0"
```

**Error:**
```
carthage bootstrap --platform ios                                                                                                              
*** Checking out specta at "v1.0.3"
*** Checking out WLXBluetoothDevice at "0.2.0"
*** Downloading Box.framework binary at "1.2.2"
*** Checking out CocoaLumberjack at "2.0.1"
*** Checking out Result at "0.4.4"
*** Checking out expecta at "v1.0.3"
*** Downloading ReactiveCocoa.framework binary at "v3.0"
*** xcodebuild output can be found in /var/folders/gf/dslbq4t94zg44sz7wnk2lg2r0000gp/T/carthage-xcodebuild.1vdEcz.log
*** Building scheme "CocoaLumberjack" in Lumberjack.xcodeproj
fatal error: SDK count 3 in scheme CocoaLumberjack is not supported: file /Users/guidomb/Documents/projects/forks/Carthage/Source/CarthageKit/Xcode.swift, line 959

Caught signal triggered by the Swift runtime!
Illegal instruction: 4

Unfortunately, this is probably a bug in Swift and not Carthage. If
this is preventing you from doing work, please file an issue and we'll
do our best to work around it:
https://github.com/Carthage/Carthage/issues/new

Please also consider filing a radar with Apple, containing the version
of Carthage and any crash report found in Console.app.


Caught signal triggered by the Swift runtime!
Illegal instruction: 4

Unfortunately, this is probably a bug in Swift and not Carthage. If
this is preventing you from doing work, please file an issue and we'll
do our best to work around it:
https://github.com/Carthage/Carthage/issues/new

Please also consider filing a radar with Apple, containing the version
of Carthage and any crash report found in Console.app.


Caught signal triggered by the Swift runtime!
Illegal instruction: 4

Unfortunately, this is probably a bug in Swift and not Carthage. If
this is preventing you from doing work, please file an issue and we'll
do our best to work around it:
https://github.com/Carthage/Carthage/issues/new

Please also consider filing a radar with Apple, containing the version
of Carthage and any crash report found in Console.app.


Caught signal triggered by the Swift runtime!
Illegal instruction: 4

Unfortunately, this is probably a bug in Swift and not Carthage. If
this is preventing you from doing work, please file an issue and we'll
do our best to work around it:
https://github.com/Carthage/Carthage/issues/new

Please also consider filing a radar with Apple, containing the version
of Carthage and any crash report found in Console.app.


Caught signal triggered by the Swift runtime!
Illegal instruction: 4

Unfortunately, this is probably a bug in Swift and not Carthage. If
this is preventing you from doing work, please file an issue and we'll
do our best to work around it:
https://github.com/Carthage/Carthage/issues/new

Please also consider filing a radar with Apple, containing the version
of Carthage and any crash report found in Console.app.
```

**Comment by guidomb** [comment]
Could be reproduced more easily by just having the following `Cartfile`

```
github "CocoaLumberjack/CocoaLumberjack" "swift_2.0"
```

Seems that CocoaLumberjack is doing something that makes Carthage crash really bad. This was working with Carthage 0.8.0

**Comment by guidomb** [comment]
The problem seems to be here https://github.com/Carthage/Carthage/blob/master/Source/CarthageKit/Xcode.swift#L932

**Comment by ikesyo** [comment]
Sorry for the inconvenience, I've submitted a pull request to fix this: #793

**Comment by abbeycode** [comment]
@guidomb that is definitely the problem line. The crash happens when `fatalError()` is called, but it's the Swift runtime that crashes. I mentioned it [in a comment on my pull request](https://github.com/Carthage/Carthage/pull/583#issuecomment-129498046). I wasn't sure what to do about it.
